drivers/windows/R/driver-linux.R

```diff
@@ -0,0 +1,46 @@
+hipercow_driver_linux <- function() {
+  linux_driver <- hipercow_driver_windows()
```

Comment:
We will regret this. Can you please split into dide_driver_base() and then a linux and a windows version that add their relevant bits